packer/ansible/roles/aggregate/files/aggregate-config.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+PROPS_FILE=/var/lib/tomcat8/webapps/ROOT/WEB-INF/classes/security.properties
+HELP=NO
+POSITIONAL=()
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    --help)
+    HELP=YES
+    shift
+    ;;
+    --fqn)
+    FQN="$2"
+    shift
+    shift
+    ;;
+    --http-port)
+    HTTP_PORT="$2"
+    shift
+    shift
+    ;;
+    --https-port)
+    HTTPS_PORT="$2"
+    shift
+    shift
+    ;;
+    *)
+    POSITIONAL+=("$1")
+    shift
+    ;;
+esac
+done
+set -- "${POSITIONAL[@]}"
+
+help() {
+  echo "Usage: aggregate-config <flags> [<arguments>...]"
+  echo ""
+  echo "Flags:"
+  echo "--help                  Show this help message"
+  echo ""
+  echo "Arguments:"
+  echo "--fqn        <value>    Set a new FQN (fully qualified name)"
+  echo "--fqn        auto       Enable automatic detection of FQN"
+  echo "                        (not recommended for production environments)"
```

Comment:
I don't think that's true. If you have DHCP, it's perfectly fine to use auto for production. Maybe instead, `Not recommended for dynamic IP addresses`